integration-tests/tests/standard_cases/rpc.rs

```diff
@@ -1,17 +1,16 @@
 //! Runs standard test cases against TestNet with several nodes running in separate threads.
 //! The communication is performed through `RPCUser` that uses the standard RPC API to communicate.
 #[cfg(feature = "expensive_tests")]
-#[cfg(test)]
 mod test {
```

Comment:
@nikurt this `mod test` here and in the other file is not necessary now. Let's move `#[cfg(feature = "expensive_tests")]` to the module declaration in the `main.rs` remove an extra level of nesting / indentation here.